comp/core/workloadmeta/collectors/external/process/process_collector.go

```diff
@@ -0,0 +1,199 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
```

Comment:
not sure about the use of `external` here but I did have to move it out of the `internal` folder so that this package could be imported directly.